README.md

```
@@ -4,8 +4,22 @@
 
 [![Globalization Ext. CI/CD Pipeline](https://github.com/codebeltnet/globalization/actions/workflows/pipelines.yml/badge.svg)](https://github.com/codebeltnet/globalization/actions/workflows/pipelines.yml) [![codecov](https://codecov.io/gh/codebeltnet/globalization/graph/badge.svg?token=BN2UhFM3bb)](https://codecov.io/gh/codebeltnet/globalization) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=globalization&metric=alert_status)](https://sonarcloud.io/dashboard?id=globalization) [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=globalization&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=globalization) [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=globalization&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=globalization) [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=globalization&metric=security_rating)](https://sonarcloud.io/dashboard?id=globalization) [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/codebeltnet/xunit/badge)](https://scorecard.dev/viewer/?uri=github.com/codebeltnet/globalization)
 
+### ℹ️ About
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)